toolkit/pythonit_toolkit/api/extensions.py

```diff
@@ -1,14 +1,15 @@
 from sentry_sdk import add_breadcrumb
 from strawberry.extensions import Extension
+from strawberry.types.execution import ExecutionContext
```

Comment:
what is this change? adding back ExecutionContext?

Reply:
Yeah, think initially I did this change here wrongly, and during the merge git reverted by mistake... 
And surprisingly the tests are working?!?! 🤔